src/event-db/migrations/V2__event_tables.sql

```diff
@@ -34,7 +34,8 @@ CREATE TABLE event
     committee_size INTEGER NOT NULL,
     committee_threshold INTEGER NOT NULL,
 
-    extra JSONB
+    extra JSONB,
+    cast_to JSONB
```

Comment:
Didn't really get the idea of this field. Can you please a little bit elaborate :)

Reply:
`cast_to` is where an event casts its votes.  The format of the embedded json is just an idea at the moment.  But I wanted to add this field while I was in there so when we want to run multiple jorm chains, or push votes to mainnet or our p2p network, we have somewhere we can configure it.

I made it an array so we can declare multiple destinations, so we can cast votes into all of them at the same time. (Ie, jorm and the p2p network).  Just by tweaking the config.

Processing of casting votes would just iterate this array and send the vote where it needs to go (or do it in parallel).

Reply:
Got it ! thanks !
But what do you think to use here an enumeration type ?
https://www.postgresql.org/docs/current/datatype-enum.html

Reply:
Those kinds of types require schema changes which I am trying to minimize.
Schema changes are a lot harder to manage than data changes within the same schema.

Reply:
For example,  we will define jormungandr as a place to cast votes first.
And we can probably guess now what we would need.
Just a "Chain ID" and the fact this is Jormungandr.
But what do we need for voting on the athena network?  No idea,  I can guess but it will probably be wrong.
But with a JsonB type it doesn't matter,  I can add an athena type later,  tools that understand it can process it, others will ignore it (or they should).  In fact testing off jsonb types needs to include unexpected data to make sure the consuming services don;t do anything crazy.  The basic schema of these json types also needs to be defined well enough that they are  unambiguous.

Reply:
As we go, I will actually create proper json schema to define what should be in these JSONB fields.

Reply:
Got it, make sense, thank you !